server/fakedb/Schedule.csv

```diff
@@ -0,0 +1,22 @@
+﻿eventId,start,end,locationLatitude,locationLongitude,repeat
```

Comment:
Also, what's "repeat" field?

Reply:
For regular meetings which are repeated over time.

Reply:
Is this a boolean field that captures whether a given event is recurring or not? Or is this the field that captures the pattern itself?

Reply:
We decided to remove the repeat field because of complexity.